README.md

```diff
@@ -1,3 +1,54 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Installation
+[Get Composer](https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md)
+
+```bash
+composer require --dev  mollie/php-coding-standards:v1.0
+```
+
+## Usage
+
+### Manual Triggering
+Run following command in your project directory, that will run fixer for every `.php` file.
+```bash
+vendor/bin/php-cs-fixer fix .
```

Comment:
Please leave out the dot, as that is a file specification, at which point the finder in a `.php_cs.dist` will be ignored.